gapic-common/Gemfile

```diff
@@ -2,5 +2,3 @@ source "https://rubygems.org"
 
 # Specify your gem's dependencies in gapic-common.gemspec
 gemspec
-
-gem "google-cloud-core", github: "googleapis/google-cloud-ruby"
```

Comment:
So `google-cloud-core` was required by `gapic-common` but not needed?

Reply:
It's a development dependency. But we no longer need the copy from github master; the latest release will do.